synapse/logging/opentracing.py

```diff
@@ -886,7 +912,7 @@ def _tag_args_inner(*args: P.args, **kwargs: P.kwargs) -> R:
         for i, arg in enumerate(argspec.args[1:]):
```

Comment:
This isn't your change, but I don't really understand the offset of 1 here. I'm guessing that we want to ignore the first arg to a function call because that arg is a `self` or `cls`. But if that's the case, shouldn't we pass `start=1` to enumerate or else use `args[i+1]`)?

Reply:
I don't know, I was a bit confused by it too but decided not to touch it.